inst/include/cpp11/declarations.hpp

```diff
@@ -6,8 +6,6 @@
 
 #ifndef CPP11_PARTIAL
 #include "cpp11.hpp"
```

Comment:
I have not found mentions of `CPP11_PARTIAL` so perhaps `"cpp11.hpp"` could be included unconditionally ?

Reply:
The point of the ifndef is so you can include only the cpp11 headers you actually use conditionally in `pkg_types.hpp` if desired. It is not the default because I didn't see much of a compilation difference one way or the other from doing so. But I think it is a useful feature to have.

Reply:
Thanks. Either way, if the user wants to `using namespace cpp11` they can do it in `{pkg}_types.hpp`